cookbook/controller/service.rst

```diff
@@ -240,22 +240,22 @@ Base Controller Methods and Their Service Replacements
 This list explains how to replace the convenience methods of the base
 controller:
 
-:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::createForm` (service: ``form.factory``)
+``createForm()`` (service: ``form.factory``)
```

Comment:
I'm curious: what's the reason for removing the links to the base `Controller` methods?